lib/profileAgents.js

```diff
@@ -8,16 +8,28 @@ import * as meterClient from './meterClient.js';
 import * as profileMeters from './profileMeters.js';
 import * as utils from './utils.js';
 import * as zcaps from './zcaps.js';
+import {CapabilityAgent, KmsClient} from '@digitalbazaar/webkms-client';
 import assert from 'assert-plus';
-import {CapabilityAgent} from '@digitalbazaar/webkms-client';
+import {EdvClient} from '@digitalbazaar/edv-client';
+import {httpsAgent} from '@bedrock/https-agent';
+import {keyResolver} from './keyResolver.js';
+import {klona} from 'klona';
 
 const {
   assertKeystoreOptions,
   createCapabilityAgent,
   getCollection,
-  getPublicAliasTemplate
+  getEdvConfig,
+  getEdvDocument,
+  getProfileSigner,
+  getPublicAliasTemplate,
+  removeSecretsFromRecords
 } = utils;
-const {util: {BedrockError}} = bedrock;
+const {config, util: {BedrockError}} = bedrock;
+
+const {
+  defaultZcapTtl: DEFAULT_PROFILE_AGENT_ZCAP_TTL,
+} = config.profile.profileAgent;
```

Comment:
This won't work properly, it must be done on a bedrock event after the config has been processed. Check other modules / files for examples.

Reply:
Updated to get the values on `bedrock-start` event - [d2333bc](https://github.com/digitalbazaar/bedrock-profile/pull/99/commits/d2333bc16ea9182633252e03de1b95bb0751a87c)